OsmAnd-java/src/main/java/net/osmand/binary/GeocodingUtilities.java

```diff
@@ -146,7 +146,7 @@ public String toString() {
 		List<GeocodingResult> lst = new ArrayList<GeocodingUtilities.GeocodingResult>();
 		List<RouteSegmentPoint> listR = new ArrayList<BinaryRoutePlanner.RouteSegmentPoint>();
 		// we allow duplications to search in both files for boundary regions 
-		rp.findRouteSegment(lat, lon, ctx, listR, false, true);
+		rp.findRouteSegment(lat, lon, ctx, listR, true, true);
```

Comment:
this parameter transportStop = true ? why do we need it ?

Reply:
It necessary in order to avoid add GPS_POSSIBLE_ERROR in the https://github.com/osmandapp/OsmAnd/blob/3d62dd2fba4207a6255632e0e755cedd92a2e4b8/OsmAnd-java/src/main/java/net/osmand/router/RoutePlannerFrontEnd.java#L177